karma.conf.js

```diff
@@ -27,6 +28,12 @@ module.exports = function (config) {
             {
                 test: /\.json$/,
                 loader: 'json'
+            },
+            {
+              test: /\.css$/,
+              exclude:/node_modules/,
+              //NOTE: Avoid import like [name]__[local]___[hash:base64:5] to create a well known class name
+              loader: ExtractTextPlugin.extract('style','css?modules&importLoaders=1&localIdentName=[local]')
```

Comment:
This is the key to create a well known css class name where we are testing.

If we use `[name]__[local]___[hash:base64:5]` like in dev environment we are going to expect unique values like `styles__pageError404__XXX` where `XXX` is the hash value and this value could be changed